migration/v0.16.x-v0.17.x/migrate-openid.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+here="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+set -euo pipefail
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+
+# Make the script executable
+"${here}/../../bin/ck8s" ops kubectl sc -n elastic-system exec opendistro-es-master-0 -- chmod +x ./plugins/opendistro_security/tools/securityadmin.sh
+# Run the script to update the configuration
+"${here}/../../bin/ck8s" ops kubectl sc -n elastic-system exec opendistro-es-master-0 -- ./plugins/opendistro_security/tools/securityadmin.sh \
+    -f plugins/opendistro_security/securityconfig/config.yml \
+    -icl -nhnv \
+    -cacert config/admin-root-ca.pem \
+    -cert config/admin-crt.pem \
+    -key config/admin-key.pem
+
+"${here}/../../bin/ck8s" ops kubectl sc delete pod opendistro-es-master-0 -n elastic-system
+"${here}/../../bin/ck8s" ops kubectl sc get pods -n elastic-system | awk '/opendistro-es-kibana/{print $1}' | xargs "${here}/../../bin/ck8s" ops kubectl sc delete -n elastic-system pod
```

Comment:
When you run apply does it trigger a statefulset upgrade?
If not i feel like this is done in the wrong order.
When you run apply, it should trigger an update of the securityconfig secret.
You then need to restart the master pod**s** so that they mount the latest version of the secret.

Reply:
Done!